charts/cp-schema-registry/templates/schema-registry-ui.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  {{- if .Values.ui.annotations }}
+  annotations: {{ toYaml .Values.ui.annotations | nindent 4 }}
+  {{- end}}
+  labels:
+    app: {{ template "cp-schema-registry.fullname" . }}
+  {{- if .Values.ui.labels }}
+{{ toYaml .Values.ui.labels | indent 4}}
+  {{- end}}
+  name: {{ template "cp-schema-registry.fullname" . }}-ui
+spec:
+  replicas: {{ .Values.ui.replicaCount }}
+  selector:
+    matchLabels:
+      app: {{ template "cp-schema-registry.fullname" . }}
+      service: schema-registry-ui
+  template:
+    metadata:
+      {{- if .Values.ui.annotations }}
+      annotations: {{ toYaml .Values.ui.annotations | nindent 8 }}
+      {{- end }}
+      labels:
+        app: {{ template "cp-schema-registry.fullname" . }}
+        service: schema-registry-ui
+      {{- if .Values.ui.labels }}
+{{ toYaml .Values.ui.labels | indent 8}}
+      {{- end}}
+    spec:
+      securityContext:
+      {{- if .Values.securityContext }}
+{{ toYaml .Values.securityContext | indent 8 }}
+      {{- end}}
```

Comment:
Same remark than before: nindent and the if should include the `securityContext` keyword
=> 
```
      {{- if .Values.securityContext }}
      securityContext: {{ toYaml .Values.securityContext | nindent 8 }}
      {{- end}}
```